lib/doctor.ts

```diff
@@ -32,6 +32,51 @@ export class Doctor implements NativeScriptDoctor.IDoctor {
 		let result: NativeScriptDoctor.IInfo[] = [];
 		const sysInfoData = await this.sysInfo.getSysInfo(config);
 
+		result = result.concat(
+			this.processSysInfoItem({
+				item: sysInfoData.gitVer,
+				infoMessage: "Git is installed and is configured properly.",
+				warningMessage: "Git is not installed or not configured properly.",
+				additionalInformation: "You will not be able to create and work with Screen Builder projects." + EOL
+					+ "To be able to work with Screen Builder projects, download and install Git as described" + EOL
+					+ "in https://git-scm.com/downloads and add the git executable to your PATH.",
+				platforms: Constants.SUPPORTED_PLATFORMS
+			})
+		);
+
+		if (config && config.platform === Constants.ANDROID_PLATFORM_NAME) {
```

Comment:
You can extend this:
```TypeScript
if (!config || !config.platform || config.platform === Constants.ANDROID_PLATFORM_NAME) {
```

You can do the same for iOS and this way you can remove the whole if on line 55